etl/steps/data/grapher/who/2024-06-17/guinea_worm.meta.yml

```diff
@@ -0,0 +1,112 @@
+
+        This is supplemented with more recent changes to Guinea worm disease certification:
+        - Angola has had endemic status since 2020:  https://www.who.int/news/item/23-09-2020-eradicating-dracunculiasis-human-cases-and-animal-infections-decline-as-angola-becomes-endemic
+        - Kenya was certified guinea worm free in 2018: https://www.who.int/news/item/21-03-2018-dracunculiasis-eradication-south-sudan-claims-interruption-of-transmission-in-humans
+        - DRC was certified guinea worm free in 2022: https://www.who.int/news/item/15-12-2022-the-democratic-republic-of-the-congo-certified-free-of-dracunculiasis-transmission-by-who"
+    origins:
+      - title: "Certification status of dracunculiasis eradication"
+        date_published: "2017"
+        producer: World Health Organization
+        attribution_short: WHO
+        citation_full: "World Health Organization. Certification status of dracunculiasis eradication, 2017"
+        url_main: https://web.archive.org/web/20211024081702/https://apps.who.int/dracunculiasis/dradata/html/report_Countries_t0.html
+        date_accessed: "2024-06-17"
+
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+dataset:
+  update_period_days: 365
+
+tables:
+  guinea_worm:
+    variables:
+      year_certified:
+        title: Year country was certified free of guinea worm disease
+        unit: ""
+        description_short: To be certified as free of guinea worm disease, a country must have reported zero indigenous cases through active surveillance for at least three consecutive years.
+        description_from_producer:
+          Elimination of dracunculiasis is the confirmed absence of the emergence of adult female worms (the interruption of transmission of Dracunculus medinensis)
+          in humans and animals for three consecutive years or longer from a country with such a low risk of reintroduction of the parasite that preventive measures
+          could be reduced to a strict minimum.
+        display:
+          numDecimalPlaces: 0
+      time_frame_certified:
+        title: Time country was certified free of guinea worm disease or status of certification
+        unit: ""
+        description_short: To be certified as free of guinea worm disease, a country must have reported zero indigenous cases through active surveillance for at least three consecutive years.
+        type: ordinal
+        sort:
+          - "1997-1999"
+          - "2000s"
+          - "2010s"
+          - "2020s"
+          - "Pre-certification"
+          - "Endemic"
+      certification_status:
+        title: Certification status over time
+        unit: ""
+        description_short:
+          The current and historical values for the status of Guinea worm disease (Dracunculiasis) as certified by the WHO.
```

Comment:
You could add the DoD in here e.g. [Guinea worm disease (Dracunculiasis)](#dod:guinea-worm)